tests/e2e_tests/subcommands/stake/test_set_revoke_child_hotkeys.py

```diff
@@ -0,0 +1,260 @@
+    assert local_chain.query("SubtensorModule", "NetworksAdded", [1]).serialize()
+
+    # Register Bob
+    bob_keypair, bob_exec_command, bob_wallet = setup_wallet("//Bob")
+
+    # Register Alice neuron to the subnet
+    alice_exec_command(
+        RegisterCommand,
+        [
+            "s",
+            "register",
+            "--netuid",
+            "1",
+        ],
+    )
+
+    # Alice to Stake to become to top neuron after the first epoch
+    alice_exec_command(
+        StakeCommand,
+        [
+            "stake",
+            "add",
+            "--amount",
+            "100000",
+        ],
+    )
+
+    # Register Bob neuron to the subnet
+
+    bob_exec_command(
+        RegisterCommand,
+        [
+            "s",
+            "register",
+            "--netuid",
+            "1",
+        ],
+    )
+
+    # Assert no child hotkeys on subnet
+    subtensor = bittensor.subtensor(network="ws://localhost:9945")
+    assert (
+        subtensor.get_children_info(netuid=1) == []
+    ), "Child hotkeys are already set on new subnet. "
+
+    # Run set child
+    # btcli stake set_child --child <child_hotkey> --hotkey <parent_hotkey> --netuid 1 --proportion 0.3
+    alice_exec_command(
+        SetChildCommand,
+        [
+            "stake",
+            "set_child",
+            "--netuid",
+            "1",
+            "--child",
+            str(bob_keypair.ss58_address),
+            "--hotkey",
+            str(alice_keypair.ss58_address),
+            "--proportion",
+            "0.3",
+            "--wait_for_inclusion",
+            "True",
+            "--wait_for_finalization",
+            "True",
+        ],
+    )
+
+    subtensor = bittensor.subtensor(network="ws://localhost:9945")
+    assert (
+        len(subtensor.get_children_info(netuid=1)[alice_keypair.ss58_address]) == 1
+    ), "failed to set child hotkey"
+
+    output = capsys.readouterr().out
+    assert "✅ Finalized" in output
+
+    # Run revoke child
+    # btcli stake revoke_child --child <child_hotkey> --hotkey <parent_hotkey> --netuid 1
+    alice_exec_command(
+        RevokeChildCommand,
+        [
+            "stake",
+            "revoke_child",
+            "--netuid",
+            "1",
+            "--child",
+            str(bob_keypair.ss58_address),
+            "--hotkey",
+            str(alice_keypair.ss58_address),
+            "--wait_for_inclusion",
+            "True",
+            "--wait_for_finalization",
+            "True",
+        ],
+    )
+
+    assert subtensor.get_children_info(netuid=1) == [], "failed to revoke child hotkey"
+
+    output = capsys.readouterr().out
+    assert "✅ Finalized" in output
+
+
+"""
+Test the set children hotkey multiple mechanism. 
+
+Verify that:
+* No children hotkeys at subnet creation
+* Subnet owner an set multiple children in one call
+* Child hotkeys are set properly with correct proportions
+"""
+
+
+def test_set_revoke_children(local_chain, capsys):
```

Comment:
Pls add at least single line docstring to the test case.

Reply:
There is a docstring above this test that explains what its function is. Do you want me to move it down?